packages/application-components/src/components/modal-pages/internals/modal-page-header-default-controls.tsx

```diff
@@ -1,16 +1,12 @@
 import React from 'react';
-import { useIntl, MessageDescriptor, IntlShape } from 'react-intl';
-import {
-  Spacings,
-  PrimaryButton,
-  SecondaryButton,
-} from '@commercetools-frontend/ui-kit';
+import { MessageDescriptor } from 'react-intl';
 import filterDataAttributes from '../../../utils/filter-data-attributes';
+import { SaveButton, CancelButton } from '../../../utils/form-buttons';
 
 type Label = string | MessageDescriptor;
 type Props = {
-  labelSecondaryButton: Label;
-  labelPrimaryButton: Label;
+  labelSecondaryButton?: Label;
```

Comment:
Shouldn't those props be required? 🤔

Reply:
Oh, actually I "abandoned" this `ModalPageHeaderDefaultControls` component because it would just become an array of (or a fragment wrapping) the buttons in [default-form-buttons.tsx](https://github.com/commercetools/merchant-center-application-kit/blob/9ddaf1a1abe817575a44d30d085109422fce520f/packages/application-components/src/utils/default-form-buttons.tsx). And the buttons there already have default labels.